MapboxMobileEventsTests/MMEAPIClientTests.mm

```diff
@@ -31,6 +31,9 @@ - (void)setupUserAgent;
     
     __block MMEAPIClient *apiClient;
     __block int64_t timeoutInNanoseconds = 1000000000;
+    __block NSURLSessionAuthChallengeDisposition receivedDisposition;
+    __block NSURLSessionAuthChallengeDisposition expectedDefaultDisposition = NSURLSessionAuthChallengePerformDefaultHandling;
```

Comment:
`__block` is only needed for references which will be assigned to (changed) within a block other than the current scope. `timeoutInNanoseconds`, `expectedDefaultDisposition`, and `expectedCancelDisposition` technically don't require `__block`